Show uncompressed data size in system.tables (#57186)
ClickHouse/ClickHouse: docs/en/operations/system-tables/tables.md

    - If the table stores data on disk, returns used space on disk (i.e. compressed).
    - If the table stores data in memory, returns approximated number of used bytes in memory.

> - `total_bytes_uncompressed` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) - Total number of uncompressed bytes, if it is possible to quickly determine exact number of bytes from checksums for the table on storage, otherwise `NULL` (does not includes any underlying storage).

Reviewer: `nit:`
```suggestion
- `total_bytes_uncompressed` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) - Total number of uncompressed bytes, if it's possible to quickly determine the exact number of bytes from the part checksums for the table on storage, otherwise `NULL` (does not take underlying storage (if any) into account).
```
